packages/blockchain-link/src/workers/solana/index.ts

```diff
@@ -10,6 +10,46 @@ export type SolanaAPI = Connection;
 type Context = ContextType<SolanaAPI>;
 type Request<T> = T & Context;
 
+const fetchTransactionPage = async (
```

Comment:
Note that this implementation fetches only a single page of tx history (not parsed), I wanted to split stuff around pagination and the actual logic for getting a page of transaction history so its easier to review. You will see changes to this logic in next commits